### PR TITLE
imdb clone react-based

### PR DESCRIPTION
**Pull request by Rahul-Maity**
This is a imdb clone using react react,react-router,jsx and an api.

### CODE REVIEW

File: src/json/Contributors.json [hunk 1]
```diff
@@ -401,6 +401,12 @@
       "avatarUrl": "https://avatars.githubusercontent.com/u/73987409?v=4",
       "github": "vk-vikas",
       "twitter": ""
+    },
+    {
+      "name":"Rahul Maity",
+      "avatarUrl":"https://avatars.githubusercontent.com/u/113348978?v=4",
+      "github":"Rahul-Maity",
+      "twitter":"Rahul Maity"
```
Comment (devvsakib) [inline src/json/Contributors.json:409]: Please use your twitter username here

**Review by devvsakib — CHANGES_REQUESTED** [review]
image
you have another repository in your project folder. Please delete this repository

To DO:
Navigate to  `PROJECTS/IMDB-CLONE`
Open terminal and run `rm -rf .git`
Then git add, commit, push

**Comment by Rahul-Maity** [comment]
i did those steps.what now

**Comment by Rahul-Maity** [comment]
showing "Everything up-to-date" in the terminal

File: src/json/Contributors.json (hunk 1 above)
Comment (Rahul-Maity) [inline src/json/Contributors.json:409]:
```suggestion
      "twitter":"mrahulmaity623"
```

**Pull request closed without merging**